autowrap/ConversionProvider.py

```diff
@@ -627,37 +636,47 @@ def input_conversion(self, cpp_type, argument_var, arg_num):
            and (not cy_tt_key.is_enum and tt_key.base_type in self.converters.names_of_wrapper_classes):
             raise Exception("Converter can not handle wrapped classes as keys and values in map")
 
+        value_conv_code = ""
+        value_conv_cleanup = ""
+        key_conv_code = ""
+        key_conv_cleanup = ""
+
+        tt_key, tt_value = cpp_type.template_args
+
         if cy_tt_value.is_enum:
             value_conv = "<%s> value" % cy_tt_value
         elif tt_value.base_type in self.converters.names_of_wrapper_classes:
             value_conv = "deref((<%s>value).inst.get())" % tt_value.base_type
+        elif tt_value in self.converters:
+            value_conv_code, value_conv, value_conv_cleanup = \
+                self.converters.get(tt_value).input_conversion(tt_value, "value", 0)
         else:
             value_conv = "<%s> value" % cy_tt_value
 
         if cy_tt_key.is_enum:
             key_conv = "<%s> key" % cy_tt_key
         elif tt_key.base_type in self.converters.names_of_wrapper_classes:
             key_conv = "deref(<%s *> (<%s> key).inst.get())" % (cy_tt_key, py_tt_key)
-
-            code = Code().add("""
-                |cdef libcpp_map[$cy_tt_key, $cy_tt_value] * $temp_var = new
-                + libcpp_map[$cy_tt_key, $cy_tt_value]()
-
-                |for key, value in $argument_var.items():
-                |   deref($temp_var)[ $key_conv ] = $value_conv
-                """, locals())
-
```

Comment:
this is a cleanup, this code has no effect, see https://github.com/uweschmitt/autowrap/blob/master/autowrap/ConversionProvider.py#L642

it get overridden by

https://github.com/uweschmitt/autowrap/blob/master/autowrap/ConversionProvider.py#L653